plugins/kubernetes-backend/src/service/KubernetesBuilder.test.ts

```diff
@@ -267,4 +271,78 @@ describe('KubernetesBuilder', () => {
       expect(response.status).toEqual(200);
     });
   });
+
+  describe('post /proxy', () => {
+    const worker = setupServer();
+    setupRequestMockHandlers(worker);
+
+    it('returns the given request body', async () => {
+      const requestBody = {
+        kind: 'Namespace',
+        apiVersion: 'v1',
+        metadata: {
+          name: 'new-ns',
+        },
+      };
+
+      const proxyEndpointRequest = request(app)
+        .post('/proxy/api/v1/namespaces')
+        .set(HEADER_KUBERNETES_CLUSTER, 'some-cluster')
+        .send(requestBody);
+
+      worker.use(
+        rest.post('https://localhost:1234/api/v1/namespaces', (req, res, ctx) =>
+          req
+            .arrayBuffer()
+            .then(body =>
+              res(
+                ctx.set('content-type', `${req.headers.get('content-type')}`),
+                ctx.body(body),
+              ),
+            ),
+        ),
```

Comment:
this part is repeated in both tests -- can we extract this into a `beforeEach`? Not sure if you can call `use` on a server multiple times...

Reply:
Done.